powerauth-backend-tests/src/test/resources/application.properties

```diff
@@ -2,11 +2,11 @@
 powerauth.rest.url=http://localhost:8080/powerauth-java-server/rest
 
 # Standard RESTful integration service URL
-powerauth.integration.service.url=http://localhost:8080/enrollment-server
+powerauth.integration.service.url=http://localhost:8081/enrollment-server
```

Comment:
Why are we changing this?

Reply:
I guess this is because of the standalone run from IDE, where a differrent port is configured. Anyway, this should not be commited.

Reply:
Apologies, this should not happen